src/qmflows/cp2k_utils.py

```diff
@@ -362,9 +364,6 @@ def _(unit: Optional[str]) -> List[str]:
         return [f'[{unit}] {{}}']
 
 
-NDFrame: type = pd.DataFrame.__bases__[0]
```

Comment:
You make me happy every time that you delete a magic method/property :+1:

Reply:
But I like magic :(